.buildkite/pipeline.yml

```diff
@@ -81,11 +81,11 @@ steps:
     if: |
       // Only run if:
       // 1. There's a tag and it's of the form 'v*' or 'operator/v*'
-      // 2. Source branch is main or release/*
+      // 2. Source branch is main or release/* (during release the branch will be equal to tag name)
```

Comment:
Can you add the URL from the PR description to this comment.  If we remove the branch checks, let's preserve the comment about branch == tag so no one forgets and tries to add it back in later :)